src/Features/VisualBasic/VBFeaturesResources.resx

```diff
@@ -1051,4 +1051,178 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
+    <value>parameter</value>
+  </data>
+  <data name="AttributeList" xml:space="preserve">
+    <value>attributes</value>
+  </data>
+  <data name="Attribute" xml:space="preserve">
+    <value>attribute</value>
+  </data>
+  <data name="TryBlock" xml:space="preserve">
+    <value>Try block</value>
+  </data>
+  <data name="CatchClause" xml:space="preserve">
+    <value>Catch clause</value>
+  </data>
+  <data name="FinallyClause" xml:space="preserve">
+    <value>Finally clause</value>
+  </data>
+  <data name="UsingStatement" xml:space="preserve">
+    <value>Using statement</value>
+  </data>
+  <data name="UsingBlock" xml:space="preserve">
+    <value>Using block</value>
+  </data>
+  <data name="WithStatement" xml:space="preserve">
+    <value>With statement</value>
+  </data>
+  <data name="WithBlock" xml:space="preserve">
+    <value>With block</value>
+  </data>
+  <data name="SyncLockStatement" xml:space="preserve">
+    <value>SyncLock statement</value>
+  </data>
+  <data name="SyncLockBlock" xml:space="preserve">
+    <value>SyncLock block</value>
+  </data>
+  <data name="ForEachStatement" xml:space="preserve">
+    <value>For Each statement</value>
+  </data>
+  <data name="ForEachBlock" xml:space="preserve">
+    <value>For Each block</value>
+  </data>
+  <data name="OnErrorStatement" xml:space="preserve">
+    <value>On Error statement</value>
+  </data>
+  <data name="ResumeStatement" xml:space="preserve">
+    <value>Resume statement</value>
+  </data>
+  <data name="YieldStatement" xml:space="preserve">
+    <value>Yield statement</value>
+  </data>
+  <data name="AwaitExpression" xml:space="preserve">
+    <value>Await expression</value>
+  </data>
+  <data name="LambdaExpression" xml:space="preserve">
+    <value>lambda</value>
+  </data>
+  <data name="WhereClause" xml:space="preserve">
+    <value>Where clause</value>
+  </data>
+  <data name="SelectClause" xml:space="preserve">
+    <value>Select clause</value>
+  </data>
+  <data name="FromClause" xml:space="preserve">
+    <value>From clause</value>
+  </data>
+  <data name="AggregateClause" xml:space="preserve">
+    <value>Aggregate clause</value>
+  </data>
+  <data name="LetClause" xml:space="preserve">
+    <value>Let clause</value>
+  </data>
+  <data name="SimpleJoinClause" xml:space="preserve">
+    <value>Join clause</value>
+  </data>
+  <data name="GroupJoinClause" xml:space="preserve">
+    <value>Group Join clause</value>
+  </data>
+  <data name="GroupByClause" xml:space="preserve">
+    <value>Group By clause</value>
+  </data>
+  <data name="FunctionAggregation" xml:space="preserve">
+    <value>function aggregation</value>
+  </data>
+  <data name="TakeWhileClause" xml:space="preserve">
+    <value>Take While clause</value>
+  </data>
+  <data name="SkipWhileClause" xml:space="preserve">
+    <value>Skip While clause</value>
+  </data>
+  <data name="OrderingClause" xml:space="preserve">
+    <value>ordering clause</value>
```

Comment:
Lines 1223 and 1226; these items start with a lower-case letter, but it feels like they should be upper-case to match the similar items, e.g., "Join clause" on line 1205.